### PR TITLE
TPM module renders

### PR DESCRIPTION
**Pull request by piotrdanio**

### CODE REVIEW

**Comment by acwiek** [comment]
Are you going to move those commits to another branch/PR as they are unverified?

**Comment by piotrdanio** [comment]
I have moved all unverified commits to the new branch - TPMpinput_v2

**Comment by acwiek** [comment]
create a new clean branch and cherry-pick commits to verify them or commit your changes in a different way
 keeping in mind the verification status of your commits

File: mkdocs.yml [hunk 1]
```diff
@@ -21,7 +21,8 @@ nav:
       - 'TPM driver instrumentation': instrumentation.md
     - 'Target TPM hardware interfaces': hardware_interfaces.md
     - 'Current issues': issues.md
-    - 'TPM modules pinout': tpm_modules_pinout.md
+    - 'TPM modules pinout': TPM_module_Pinout.md
```
Comment (acwiek) [inline mkdocs.yml:24]: TPM_module_Pinout.md - use snake case file name convetion

File: docs/TPM_renders.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+## The TPM concept
+According to the many varied pinouts necessary was design the concept of TPM 
+module which can fit the motherboard pinout. There was the consideration of 
+two kinds of the interface (SPI and LPC). The whole setup would be made of two 
+PCBs. One is the main PCB with the MCU and SPI or LPC configured socket and the 
+second is the adapter that fits the motherboards pinouts.
+![Flash](images/renders/koncepcja_TPM_adapter.svg)
+
+## The main PCB
```
Comment (acwiek) [inline docs/TPM_renders.md:9]: remove this paragraph as it contains outdated renders

File: docs/TPM_renders.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+<center>![Flash](images/renders/8.png)</center>
+
+## Assembly
+According to the adapter PCB, we consider a straight or angled connector. The 
+size of PCBs is indicative. An important assumption is to cut the protruding THT
+pins.
+<center>![Flash](images/renders/9.png)</center>
+<center>![Flash](images/renders/7.png){: style="height:425px"}</center>
+
+## Example of connection with motherboar
```
Comment (acwiek) [inline docs/TPM_renders.md:33]: remove this paragraph - outdated connectors

**Comment by piotrdanio** [comment]
This commits are corrected and moved to the new MR.

**Pull request closed without merging**